feedthefox/users/adapters.py

```diff
@@ -10,6 +11,68 @@
 from feedthefox.users.mozillians import MozilliansClient
 
 
+def populate_user_data(user, sociallogin, mozillian_profile=None):
+    """Populates user instance with data gathered from mozillians.org."""
+    mozillian_attrs = ['country', 'photo', 'ircname', 'city']
+
+    if mozillian_profile:
+        user.mozillian_username = mozillian_profile['username']
+        for attr in mozillian_attrs:
+            if mozillian_profile[attr].get('privacy') == 'Public':
+                setattr(user, attr, mozillian_profile[attr].get('value'))
+        if mozillian_profile['full_name']['privacy'] == 'Public':
+            full_name = mozillian_profile['full_name']['value'].split(' ', 1)
+            try:
+                first_name, last_name = full_name
+            except ValueError:
+                first_name = full_name[0]
+                last_name = ''
+        else:
+            first_name = 'Anonymous'
+            last_name = 'Mozillian'
+        user.first_name = first_name
+        user.last_name = last_name
+
+        # Add alternate emails
+        mozillian_emails = []
+        mozillian_emails.append(mozillian_profile['email'].get('value'))
+        for email_addr in mozillian_profile['alternate_emails']:
+            mozillian_emails.append(email_addr['email'])
+
+        # Lets remove the emails that we already have
+        social_email_addresses = [addr.email for addr in sociallogin.email_addresses]
+        missing_emails = list(set(mozillian_emails) ^ set(social_email_addresses))
+        for email in missing_emails:
+            sociallogin.email_addresses.append(EmailAddress(user=user,
+                                                            email=email,
+                                                            verified=True))
+    else:
+        user.is_active = False
+
+    return user
+
+
+def get_mozillian_profile(emails):
+    """Return a profile from mozillians.org give a list of emails."""
+
+    # Check that settings for mozillians.org API are present
+    if not settings.MOZILLIANS_API_URL or not settings.MOZILLIANS_API_KEY:
+        raise ImproperlyConfigured('No API_KEY or URL configured. '
+                                   'Please provide MOZILLIANS_API_URL and '
+                                   'MOZILLIANS_API_KEY in the .env file')
```

Comment:
Please ommit ``in the .env file``. Configuration can be anything that changes the host ENV vars.